pkgs/tools/text/xml/xmloscopy/default.nix

```
@@ -5,8 +5,8 @@ fzf, coreutils, libxml2, libxslt, jing, findutils, gnugrep, gnused,
 docbook5
 }:
 stdenv.mkDerivation rec {
-  name = "xmloscopy-${version}";
-  version = "0.1.2";
+  pname = "xmloscopy";
+  version = "0.1.3";
 
   buildInputs = [
     makeWrapper
```

not part of your changes, but I think this is "more correct". Doesn't change closure size though, so I guess it doesn't really matter.
```suggestion
  nativeBuildInputs = [  makeWrapper ];
```